### PR TITLE
Start hero section on website

### PR DESCRIPTION
**Pull request by bkase**
The hero section on the website is starting to look sort of like what it should look like based on the spec. I fully fleshed out the color/font styling from zeplin, but did not do layout completely (if the graphics change we may want to change how this bit is laid out)

I also hacked IBM plex fonts in by copying some styles from the legacy site. It looks like we're missing some of the weights, so we'll have to add those and we should clean up my hack.

Screen Shot 2019-03-17 at 10 56 09 PM

### CODE REVIEW

**Comment by psteckler** [comment]
"Fixed" sounds like something is broken. Maybe "Constant size" ?

File: frontend/website/src/HeroSection.re [hunk 1]
```diff
@@ -0,0 +1,131 @@
+open Css;
+open Style;
```
Comment (Schmavery) [inline frontend/website/src/HeroSection.re]: I wonder if we should avoid doing this at the top level, those modules define a lot of identifiers

File: frontend/website/src/HeroSection.re [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+               " from any device, in less data than a few tweets.",
+             )}
+          </span>
+        </p>
+      </div>,
+  };
+};
+
+module Graphic = {
+  module Placholder = {
```
Comment (Schmavery) [inline frontend/website/src/HeroSection.re]: Typo? though it being temporary, wouldn't really matter

File: frontend/website/src/HeroSection.re [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                style([color(textColor), fontWeight(`medium)]),
+              ])}>
+              {ReasonReact.string(name)}
+            </h3>
+            <h3
+              className={merge([
+                H3.basic,
+                style([
+                  color(textColor),
+                  fontWeight(sizeEmphasis ? `bold : `normal),
```
Comment (Schmavery) [inline frontend/website/src/HeroSection.re]: I guess we're planning to use this somewhere else? (Don't see sizeEmphasis set to true anywhere)

File: frontend/website/src/LegacyPage.re [hunk 4]
```diff
@@ -75,6 +75,63 @@ module Header = {
...
+          dangerouslySetInnerHTML={
+            "__html": {|
+@font-face {
+  font-family: 'IBM Plex Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url("/static/font/IBMPlexSans-Regular-Latin1.woff2") format("woff2"),
+    url("/static/font/IBMPlexSans-Regular-Latin1.woff") format("woff");
+  unicode-range: U+0000, U+000D, U+0020-007E, U+00A0-00A3, U+00A4-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2013-2014, U+2018-201A, U+201C-201E, U+2020-2022, U+2026, U+2030, U+2039-203A, U+2044, U+2074, U+20AC, U+2122, U+2212, U+FB01-FB02;
+}
```
Comment (Schmavery) [inline frontend/website/src/LegacyPage.re]: Maybe bs-css "global" css rules will work here
```reason
Css.(
  global("body", [margin(px(0))])
);
```

File: frontend/website/src/LegacyPage.re [hunk 5]
```diff
@@ -103,11 +160,11 @@ module Footer = {
...
-          {last ?
-             <span className="dn" /> :
-             <span className="f6 silver">
-               {ReasonReact.string({js| · |js})}
-             </span>}
+          {last
+             ? <span className="dn" />
+             : <span className="f6 silver">
+                 {ReasonReact.string({js| · |js})}
+               </span>}
```
Comment (Schmavery) [inline frontend/website/src/LegacyPage.re]: Looks like that opam refmt version strikes again. I have an idea to help with this, will talk later

**Pull request merged**